docker/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '2'
+
+services:
+  zookeeper:
+    image: zookeeper
+    hostname: zookeeper
+    ports:
+      - "2181:2181"
+  bookkeeper:
+    image: apache/bookkeeper
+    build: .
+    depends_on:
+      - zookeeper
+    links:
+      - zookeeper
+    environment:
+      - JAVA_HOME=/usr/lib/jvm/jre-1.8.0
+      - BK_zkServers=zookeeper:2181
+      - BK_zkLedgersRootPath = /ledgers
+  dice:
+    image: caiok/bookkeeper-tutorial
+    depends_on:
```

Comment:
seems as above comments, need more work on 'depends_on'

Reply:
I added a script to wait for bookkeeper to the dice container, it's explained in #328

Reply:
Cool. How is your verification on this? It should be OK to run dice now.

Reply:
it works now with the new docker image for the dice with the script to wait for bookkeeper and using 3 bookkeepers.

Reply:
👍

Reply:
Oh, seems you already did that in  https://github.com/caiok/bookkeeper-tutorial/pull/1. great work.